docker-compose.yml

```diff
@@ -31,8 +34,9 @@ services:
       - ./packages/common:/opt/app/node_modules/@pins/common
       - ./packages/api:/opt/app
     command: npm run start:dev
-  mongodb:
-    container_name: mongo-db
-    image: mongo:3.6.21
-    ports:
-      - 4000:27017
+    networks:
+      - appeal-planning-decision_default
+
+networks:
+  appeal-planning-decision_default:
+    external: true
```

Comment:
By linking this file by the network with the docker-compose file from the Appeals Service, this presumably means that it has a dependency on the Appeals Service file and both need to at the same time.

Perhaps this is inevitable as we continue to develop both apps but as they are currently separate entities in different repos it not necessarily immediately obvious that both need to be running.

Perhaps we need to have a wider discussion about how we best handle that, for example to we merge the docker-compose files and have a script that pulls both repos from GitHub, or something similar? 

I think at the least we should have some documentation in the README of this repo to detail this in terms of how to run the app.

@rcmsoftwaredevelopment @AnthonyJWalsh Any thoughts?

Reply:
I think this dependency is probably acceptable. Realistically our only options to mitigate this would be to set up a mock document service within the back-office repo (time, doesnt give us anything really helpful), or to go back to a mono-repo (has its own problems). Happy to chat about it though.